packages/flutter_tools/templates/template_manifest.json

```diff
@@ -98,13 +98,13 @@
         "templates/app_shared/macos.tmpl/Runner.xcworkspace/contents.xcworkspacedata",
         "templates/app_shared/macos.tmpl/Runner.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist",
         "templates/app_shared/macos.tmpl/Runner/AppDelegate.swift",
-        "templates/app_shared/macos.tmpl/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_1024.png",
-        "templates/app_shared/macos.tmpl/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_128.png",
-        "templates/app_shared/macos.tmpl/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_16.png",
-        "templates/app_shared/macos.tmpl/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_256.png",
-        "templates/app_shared/macos.tmpl/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_32.png",
-        "templates/app_shared/macos.tmpl/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_512.png",
-        "templates/app_shared/macos.tmpl/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_64.png",
+        "templates/app_shared/macos.tmpl/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_1024.png.img.tmpl",
```

Comment:
can you add a comment on line 3 explaining the .img.tmpl stuff? e.g. telling people where to find the assets

Reply:
Ah will do -- this hit me too. I found a comment in template.dart in the tool, which wasn't the first place I looked :)

Reply:
Done.